components/script/script_module.rs

```diff
@@ -1895,6 +1901,7 @@ pub(crate) fn fetch_inline_module_script(
         compiled_module.handle_mut(),
         true,
         can_gc,
+        Some(c"inlineScript"),
```

Comment:
nit: We should probably define this as a global const somewhere.

Reply:
not sure if script::script_runtime is the best place for it, but done in afb32dcc375fc537b4de8471f5dc29aeef8a88ab